src/azure-cli/azure/cli/command_modules/aro/_rbac.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import uuid
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.profiles import get_sdk
+from azure.cli.core.profiles import ResourceType
+from msrestazure.tools import resource_id
+
+
+CONTRIBUTOR = 'b24988ac-6180-42a0-ab88-20f7382dd24c'
+DEVELOPMENT_CONTRIBUTOR = 'f3fe7bc1-0ef9-4681-a68c-c1fa285d6128'
```

Comment:
Is this secret available related to some Azure resoruce?

Reply:
`CONTRIBUTOR` is Principal, assigned to this RP/Service, and is static across all the Azure. It is used for RP to get access into customer "BYO-VNET" from `vnet_resource_group_name`.

Reply:
Is there any risk that user could have change to manipulate something maliciously by using this?

Reply:
This is Azure built in role. These should not change. https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#general . As per my understanding, these should be static in AAD. Is this not a case?

```
{
    "assignableScopes": [
      "/"
    ],
    "description": "Lets you manage everything except access to resources.",
    "id": "/subscriptions/225e02bc-43d0-43d1-a01a-17e584a4ef69/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c",
    "name": "b24988ac-6180-42a0-ab88-20f7382dd24c",
    "permissions": [
      {
        "actions": [
          "*"
        ],
        "dataActions": [],
        "notActions": [
          "Microsoft.Authorization/*/Delete",
          "Microsoft.Authorization/*/Write",
          "Microsoft.Authorization/elevateAccess/Action",
          "Microsoft.Blueprint/blueprintAssignments/write",
          "Microsoft.Blueprint/blueprintAssignments/delete"
        ],
        "notDataActions": []
      }
    ],
    "roleName": "Contributor",
    "roleType": "BuiltInRole",
    "type": "Microsoft.Authorization/roleDefinitions"
  },

```
P.S ACS, APPService, IOT and few other modules are taking the same aproach.

Reply:
As for as I know, there is no hard code this string in CLI:

![image](https://user-images.githubusercontent.com/14357159/78632584-fc067100-78d1-11ea-931b-2618fb20bb47.png)

Could you get this string through graphrbac SDK like `az role`?